plugin/cr/src/main/java/org/opencds/cqf/ruler/cr/TerminologyServerClient.java

```diff
@@ -0,0 +1,79 @@
+package org.opencds.cqf.ruler.cr;
+
+import ca.uhn.fhir.context.FhirContext;
+import ca.uhn.fhir.rest.client.api.IGenericClient;
+import ca.uhn.fhir.rest.client.interceptor.AdditionalRequestHeadersInterceptor;
+import org.apache.commons.lang3.StringUtils;
+import org.hl7.fhir.r4.model.Parameters;
+import org.hl7.fhir.r4.model.ValueSet;
+import org.jetbrains.annotations.NotNull;
+import org.opencds.cqf.cql.evaluator.fhir.util.Canonicals;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+
+public class TerminologyServerClient {
+
+	private final FhirContext ctx;
+
+	private String username;
+
+	public String getUsername() {
+		return username;
+	}
+
+	public void setUsername(String username) {
+		this.username = username;
+	}
+
+	private String apiKey;
+
+	public String getApiKey() {
+		return apiKey;
+	}
+
+	public void setApiKey(String apiKey) {
+		this.apiKey = apiKey;
+	}
```

Comment:
username / ApiKey setters could be private? ApiKey getter probably should be private?